packages/svelte/src/internal/client/runtime.js

```diff
@@ -597,35 +572,38 @@ function recursively_collect_effects(
  * @param {import('./types.js').Effect} effect
  * @param {number} filter_flags
  * @param {boolean} [shallow]
- * @returns {import('./types.js').Effect[]}
+ * @returns {void}
  */
-function get_nested_effects(effect, filter_flags, shallow = false) {
-	/**
-	 * @type {import("./types.js").Effect[]}
-	 */
-	var render_effects = [];
+function flush_nested_effects(effect, filter_flags, shallow = false) {
 	/**
 	 * @type {import("./types.js").Effect[]}
 	 */
```

Comment:
```suggestion
	/** @type {import('#client').Effect[]} */
```

Reply:
(there's a few of these, up to you whether you want to just leave them for now and update them en masse later)